src/test/java/Homework15.java

```diff
@@ -0,0 +1,45 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+
+public class Homework15 {
+    @Test
+    public static void Homework15solution() throws InterruptedException {
+        WebDriver driver = new ChromeDriver();
+        try {
```

Comment:
you don't no use try catch becase he be in method `public static void Homework15solution() throws InterruptedException {`